package.json

```diff
@@ -56,6 +56,7 @@
     "isml-linter": "^5.22.1",
     "jquery": "^3.4.1",
     "popper.js": "^1.14.6",
+    "slick": "^1.12.2",
```

Comment:
I don't think this is the right slick... And, doesn't seem to me that you're are using this installation at all.

Reply:
So, I tried to install it through npm but it was not working.